### PR TITLE
Add support for firewalls

### PR DESCRIPTION
**Pull request by fnkr**
Yesterday Hetzner introduced firewalls for Hetzner Cloud.

Implementation is similar to volumes and networks. Firewalls and rules can be created using the Cloud Console or API. The Docker Machine Driver will apply firewalls specified via `--hetzner-firewalls` or `HETZNER_FIREWALLS` to new servers.

* Announcement: https://www.hetzner.com/news/03-21-firewall-beta/
* API documentation: https://docs.hetzner.cloud/#firewalls

### CODE REVIEW

File: driver.go [hunk 1]
```diff
@@ -340,6 +350,19 @@ func (d *Driver) Create() error {
 	}
 	srvopts.Networks = networks
 
+	firewalls :=  []*hcloud.ServerCreateFirewall{}
```
Comment (mxschmitt) [inline driver.go:353]:
```suggestion
	firewalls := []*hcloud.ServerCreateFirewall{}
```

**Review by mxschmitt — APPROVED** [review]
LGTM but please run gofmt

**Comment by fnkr** [comment]
@mxschmitt Done.

**Comment by JonasProgrammer** [comment]
LGTM

**Pull request merged**